### PR TITLE
Add flake8 in CI

### PR DESCRIPTION
**Pull request by k9845**
- Add flake8

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -29,7 +29,6 @@ x-server: &base-server-config
     AWS_S3_SECRET_ACCESS_KEY: ${AWS_S3_SECRET_ACCESS_KEY:-}
     AWS_S3_BUCKET_NAME: ${AWS_S3_BUCKET_NAME}
     AWS_S3_REGION_NAME: ${AWS_S3_REGION_NAME}
-    USE_S3: ${USE_S3:-}
```
Comment (ranjan-stha) [inline docker-compose.yml:32]: @k9845  Why did you remove this one? I think this is in use.

Comment (k9845) [reply docker-compose.yml:32]: Defined twice.

**Review by ranjan-stha — APPROVED** [review]
LGTM
